src/screens/Home/Home.tsx

```diff
@@ -37,6 +38,70 @@ interface HomeProps {
 
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
+  useEffect(() => {
+    proccessData();
+  }, []);
+
+  const [outStandingPayments, setOutstandingPayments] = useState<number>();
+  const [toCharge, setToCharges] = useState<number>();
+  const [customerDetails, setCustomerDetails] = useState<number>();
+  const [unpaidReports, setUnpaidReports] = useState<number>();
+
+
+
+  const proccessData = () => {
+    calculateCustomerData(currentSite);
+    calculateUnpaidReports(currentSite)
+  }
+
+  const calculateCustomerData = (ss: SiteRecord) => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let toCharge = 0;
+    let outstanding = 0;
+    //get day
+    const today = new Date();
+    const currMonth = today.getUTCMonth();
```

Comment:
We should use the moment library for date handling. It'd be good to write whatever util functions you need in `momentUtils` to add a layer of abstraction. See Tiffany's PR https://github.com/calblueprint/meepanyar/pull/21 for a good example